e3sm_diags/derivations/acme.py

```diff
@@ -1742,6 +1759,56 @@ def cosp_histogram_standardize(cld: "FileVariable"):
             (("Mass_pom",), rename),
         ]
     ),
+    # total aerosol number concentration (#/CC)
+    "a_num": OrderedDict(
+        [
+            (("cpc",), rename),
+            # Aerosol concentration from Aitken, Accumu., and Coarse mode
+            (
+                (
+                    "num_a1",
+                    "num_a2",
+                    "num_a3",
+                ),
+                lambda a1, a2, a3: a_num_sum(a1 + a2 + a3),
+            ),
+        ]
+    ),
+    # total so4 mass concentration (ng/m3)
+    "so4_mass": OrderedDict(
+        [
+            (("sulfate",), rename),
+            # Aerosol concentration from Aitken, Accumu., and Coarse mode
```

Comment:
Note: this only takes Aitken and Accumu. mode. remove Coarse in the comment

Reply:
@lannyzxj Cheng and I are working on clearing up some documentation in preparation for E3SM Tutorial. When working on ARM Diags, I noticed that when computing so4 mass concentration, only Aitken and Accumu. mode are taking account. Could you remind me why is the case? Is it to make the model data comparable to obs?